aptos-move/framework/aptos-framework/sources/delegation_pool.move

```diff
@@ -6,7 +7,7 @@ proportionally to their stake and provided the same stake-management API as the
 The main accounting logic in the delegation pool contract handles the following:
 1. Tracks how much stake each delegator owns, privately deposited as well as earned.
 Accounting individual delegator stakes is achieved through the shares-based pool defined at
-<code>aptos_std::pool_u64</code>, hence delegators own shares rather than absolute stakes into the delegation pool.
+<code>aptos_std::pool_u64<ode>, hence delegators own shares rather than absolute stakes into the delegation pool.
```

Comment:
This is a bug

Reply:
we already fixed it https://github.com/movebit/movefmt/commit/28a55ff52096655e835e6ad748b0ee23a328716f

Reply:
https://github.com/movebit/movefmt/commit/28a55ff52096655e835e6ad748b0ee23a328716f